src/lib/holocene/select/custom-option.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts" context="module">
```

Comment:
I've seen a few of these, I think it'd be good to add some docs explaining why they are necessary. You can created a `custom-option.story.md` and it will get picked up and displayed in Histoire.

Reply:
Deleted this